Kuroba/core-settings/src/main/java/com/github/k1rakishou/ChanSettings.java

```diff
@@ -376,6 +377,7 @@ private static void initInternal() {
             enableReplyFab = new BooleanSetting(provider, "preference_enable_reply_fab", true);
             captchaOnBottom = new BooleanSetting(provider, "captcha_on_bottom", true);
             neverShowPages = new BooleanSetting(provider, "never_show_page_number", false);
+            enableDraggableScrollbars = new BooleanSetting(provider, "hide_scrollbars", true);
```

Comment:
I guess this should be called `enable_draggable_scrollbars` or something like that?

Reply:
Oh shoot. Forgot to refactor this. All good now